Makefile

```diff
@@ -1,20 +1,26 @@
 build-dev:
 	docker build -t openslides-auth-dev -f Dockerfile.dev .
 
-run-dev: | build-dev
-	docker-compose -f docker-compose.dev.yml up
+run-bash: | build-dev
+	docker-compose -f docker-compose.dev.yml up -d
+	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
+	docker-compose -f docker-compose.dev.yml exec auth bash
+	docker-compose -f docker-compose.dev.yml down
+
+stop-run:
 	docker-compose -f docker-compose.dev.yml down
 
 build-prod:
 	docker build -t openslides-auth -f Dockerfile .
 
-run-prod: | build-prod
-	docker-compose -f docker-compose.prod.yml up
-	docker-compose -f docker-compose.prod.yml down
-
-build-test:
-	docker build -t openslides-auth-test -f Dockerfile.test .
+run-test: | build-dev
+	echo "########################################################################"
```

Comment:
Add ` @` before all echos (`@echo`): This supresses make to print the command, so the message is not doubled.